Dockerfile

```diff
@@ -68,6 +68,7 @@ ENV APP_USER=node
 ENV APP_HOME=/app
 ENV PORT=3000
 ENV NGINX_CONF_PATH=/etc/nginx/nginx.conf
+ENV REACT_APP_BASE_URL=${REACT_APP_BASE_URL:-''}
```

Comment:
This sets a default, to be used when the app stands alone (i.e. without basename)